plugins/discord-rpc/back.js

```diff
@@ -0,0 +1,41 @@
+const DiscordRPC = require('discord-rpc');
```

Comment:
nit: would have just called the plugin `discord` (rpc being an implementation detail, it might not be necessary to have it in the plugin name)

Reply:
Like this?

Reply:
Ah no, sorry for the confusion, I was talking about the filename `plugins/discord-rpc/back.js` -> `plugins/discord/back.js`! The code looks good 👍